references/detection/coco_utils.py

```diff
@@ -196,12 +192,15 @@ def convert_to_coco_api(ds):
 
 
 def get_coco_api_from_dataset(dataset):
+    # FIXME: This is... awful?
```

Comment:
Yeah. Happy for you to address it here, but not required.

Reply:
I would if I knew what to do lol. (I'm gonna leave this out for now I think)